rules/windows/defense_evasion_whitespace_padding_in_command_line.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2021/07/30"
+maturity = "production"
+updated_date = "2021/07/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution events where the command line value contains a long sequence of whitespace characters. Attackers
+may attempt to evade signature-based detections by padding their malicious command with unnecessary whitespace
+characters. These observations should be investigated for malicious behavior.
```

To reflect the suggested query expansion below
```suggestion
Identifies process execution events where the command line value contains a long sequence of whitespace characters or 
multiple occurrences of contiguous whitespace. Attackers may attempt to evade signature-based detections by padding 
their malicious command with unnecessary whitespace characters. These observations should be investigated for malicious 
behavior.
```